.appveyor.yml

```diff
@@ -115,34 +115,25 @@ environment:
     - FLAVOR: mingw32
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       ARCH: i686
-      B2_ADDRESS_MODEL: address-model=32
+      B2_ADDRESS_MODEL: 32
       B2_CXXSTD: 03,11
       SCRIPT: ci\appveyor\mingw.bat
-      B2_VARIANT: variant=debug
+      B2_VARIANT: debug
 
     - FLAVOR: mingw64
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       ARCH: x86_64
-      B2_ADDRESS_MODEL: address-model=64
+      B2_ADDRESS_MODEL: 64
       B2_CXXSTD: 11,17
       B2_DEFINES: define=__USE_ISOC99
       SCRIPT: ci\appveyor\mingw.bat
 
 install:
-  - set SELF=%APPVEYOR_PROJECT_NAME:-=_%
   - git clone https://github.com/boostorg/boost-ci.git C:\boost-ci-cloned
   - xcopy /s /e /q /i /y C:\boost-ci-cloned\ci .\ci
   - rmdir /s /q C:\boost-ci-cloned
   - ci\appveyor\install.bat
 
 build: off
 
-test_script:
-  - set SELF=%APPVEYOR_PROJECT_NAME:-=_%
-  - PATH=%ADDPATH%%PATH%
-  # The definition of B2_TOOLCXX omits B2_CXXSTD= if it was not defined above
-  - IF NOT DEFINED B2_CXXSTD (SET B2_TOOLCXX=toolset=%B2_TOOLSET%) ELSE (SET B2_TOOLCXX=toolset=%B2_TOOLSET% cxxstd=%B2_CXXSTD%)
-  # Echo the complete build command to the build log
-  - IF NOT DEFINED SCRIPT                         (ECHO b2 libs/%SELF:\=/% %B2_TOOLCXX% %B2_CXXFLAGS% %B2_DEFINES% %B2_THREADING% %B2_ADDRESS_MODEL% %B2_LINK% %B2_THREADING% %B2_VARIANT% -j3)
-  # Now go build...
-  - IF DEFINED SCRIPT (call libs\%SELF%\%SCRIPT%) ELSE (b2 libs/%SELF:\=/% %B2_TOOLCXX% %B2_CXXFLAGS% %B2_DEFINES% %B2_THREADING% %B2_ADDRESS_MODEL% %B2_LINK% %B2_THREADING% %B2_VARIANT% -j3)
+test_script: ci\build.bat
```

Comment:
I just wonder if PowerShell preference over BAT wouldn't simplify scripts, in general

Reply:
PowerShell is weird IMO and we have install.bat already which we need to keep (or can we call a PS from a BAT?)

Reply:
For now our BATs seem simple enough